java-engine/src/main/java/io/getunleash/engine/UnleashEngine.java

```diff
@@ -63,9 +63,10 @@ public UnleashEngine(List<IStrategy> customStrategies, IStrategy fallbackStrateg
                                         }
                                         return true;
                                     }),
-                    fallbackStrategy);
+                    fallbackStrategy, new HashSet<String>(builtInStrategies));
         } else {
-            this.customStrategiesEvaluator = new CustomStrategiesEvaluator(Stream.empty(), fallbackStrategy);
+            this.customStrategiesEvaluator = new CustomStrategiesEvaluator(Stream.empty(), fallbackStrategy,
+                    new HashSet<String>());
```

Comment:
This code path doesn't really matter for custom strategies since they'll be ignored